src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -22,6 +22,7 @@ public CommandResult execute() {
                 + "\n" + FindCommand.MESSAGE_USAGE
                 + "\n" + ListCommand.MESSAGE_USAGE
                 + "\n" + ViewCommand.MESSAGE_USAGE
+                + "\n" + EditCommand.MESSAGE_USAGE
```

Comment:
This is good, make sure to update your `help` command as well, so that internally it also reflects the new commands to the user.